pages/index.vue

```diff
@@ -59,7 +63,7 @@
 
 <script setup lang="ts">
 import { DateTime, Interval } from "luxon";
-import { useStorage } from "@vueuse/core";
+import { get, useStorage } from "@vueuse/core";
```

Comment:
a mistake of auto-import? I didn't see where `get` is used.

Reply:
Maybe